src/main/java/redis/clients/jedis/executors/ClusterCommandExecutor.java

```diff
@@ -148,7 +149,8 @@ private static long getBackoffSleepMillis(int attemptsLeft, Instant deadline) {
       throw new JedisClusterOperationException("Cluster retry deadline exceeded.");
     }
 
-    return millisLeft / (attemptsLeft * (attemptsLeft + 1));
+    long maxBackOff = millisLeft / (attemptsLeft * (attemptsLeft + 1));
```

Comment:
```suggestion
    long maxBackOff = millisLeft / (attemptsLeft * attemptsLeft);
```

Reply:
@sazzad16 
Glad to hear :)
Regarding the two possible equations A and B.
I am OK with either